addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js

```diff
@@ -78,7 +78,7 @@ export class SpreadsheetDashboardAction extends Component {
         });
```

Comment:
probably not your fault, but vertical text creates one of the ugliset font I ever saw 
![image](https://github.com/odoo/odoo/assets/31045223/84ed584d-1270-42dc-8d49-b5a828e94827)